doc/src/site/sphinx/tutorials/sw_automl.rst

```diff
@@ -0,0 +1,163 @@
+
+        .. code:: scala
+
+            automl.setSortMetric("AUC")
+            val model = automl.fit(trainingDF)
+
+        You can also get raw model details by calling the *getModelDetails()* method available on the estimator as:
+
+        .. code:: scala
+
+            model.getModelDetails()
+
+        Run Predictions
+
+        .. code:: scala
+
+            model.transform(testingDF).show()
+
+
+    .. tab-container:: Python
+        :title: Python
+
+        First, let's start PySparkling Shell as
+
+        .. code:: shell
+
+            ./bin/pysparkling
+
+        Start H2O cluster inside the Spark environment
+
+        .. code:: python
+
+            from pysparkling import *
+            hc = H2OContext.getOrCreate(spark)
+
+        Parse the data using H2O and convert them to Spark Frame
+
+        .. code:: python
+
+            import h2o
+            frame = h2o.import_file("https://raw.githubusercontent.com/h2oai/sparkling-water/master/examples/smalldata/prostate/prostate.csv")
+            sparkDF = hc.as_spark_frame(frame)
+            sparkDF = sparkDF.withColumn("CAPSULE", sparkDF.CAPSULE.cast("string"))
+            [trainingDF, testingDF] = sparkDF.randomSplit([0.8, 0.2])
+
+        Create a H2OAutoML instance and configure it according your use case via provided setters or named constructor parameters.
+        If feature columns are not specified explicitly, all columns excluding label, fold, weight and ignored columns are considered as features.
+
+        .. code:: python
+
+            from pysparkling.ml import H2OAutoML
+            automl = H2OAutoML(labelCol="CAPSULE", ignoredCols=["ID"])
+
+        By default, AutoML goes through a huge space of H2O algorithms and their hyper-parameters which requires some time. If you wish to speed up
+        the training phase, you can exclude some H2O algorithms and limit the number of trained models.
+
+        .. code:: python
+
+            automl.setExcludeAlgos(["GLM"])
+            automl.setMaxModels(10)
+
+        Train the AutoML model. The training phase returns the best model according to the sortMetric. If it's not specified, the sortMetric is chosen automatically.
+
+        .. code:: python
+
+            automl.setSortMetric("AUC")
+            model = automl.fit(trainingDF)
+
+        You can also get raw model details by calling the *getModelDetails()* method available on the estimator as:
+
+        .. code:: python
+
+            model.getModelDetails()
+
+        Run Predictions
+
+        .. code:: python
+
+            model.transform(testingDF).show()
+
+
+
+Enabling XGBoost Models when Running Sparkling Water on YARN
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The multi-node XGBoost algorithm is considered as an experimental feature of AutoML. Thus the XGBoost algorithm is disabled for AutoML by default when running
+Sparkling Water on YARN or a local cluster.
```

Comment:
XGboost is disabled just when running sparkling water in distributed env. XGboost is enabled  in `local` mode